src/custom/components/AccountDetails/Transaction/styled.ts

```diff
@@ -227,6 +227,8 @@ export const StatusLabelWrapper = styled.div`
   `};
 `
 
+// TODO: Consolidate status label logic with StatusItem
```

Comment:
Added a note to use the more recent status label style logic, that is in use for the limit orders table.